app.js

```diff
@@ -0,0 +1,205 @@
+"use strict";
+
+(async () => {
+    const getData = url => fetch(url).then(response => response.json());
+
+    const userInput = document.getElementById('search-area');
+    const searchForm = document.getElementById('search-form');
+    const allBtn = document.getElementById('all-btn');
+    const statsDiv = document.getElementById('stats');
+    const errorDiv = document.getElementById('error');
+
+    const showError = (message) => {
+        errorDiv.textContent = message;
+        errorDiv.style.display = 'block';
```

Comment:
it needs to be reset in certain events:
<img width="438" height="553" alt="Screenshot 2025-08-12 at 15 50 47" src="https://github.com/user-attachments/assets/8cb4b593-667c-4b2e-82c2-eccb31b95d4a" />